functions.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+function createAllDisplayCards(array $results): string {
```

Comment:
Put the { on the next line, i.e.
```
function createAllDisplayCards(array $results): string
{
```

Reply:
Checked all of these and amended if needed.